client/src/pages/Questions/index.js

```diff
@@ -47,10 +50,18 @@ const useStyles = makeStyles(theme => ({
 
 const  Page4 = (props) => {
 
-  const { questions } = props;
+  useEffect(() => {
+    axios('/api/v1/questions')
+      .then(res => {console.log('eeeeeeeee', res);
+       getQuestions(res.data.data)})
+      .catch(res => {})
+  }, [])
+
   const [answer1, setAnswer1] = useState("");
   const [answer2, setAnswer2] = useState("");
   const [answer3, setAnswer3] = useState("");
+  //44444444444444444444444444444444444444444444444444444444
+  const [questions, getQuestions] = useState([]);
```

Comment:
think it's a typo but could you change it from getQuestions to setQuestions

Reply:
> I was thinking would it be better to do the get questions request in a useEffect in app.js and store them in a state and pass them down as props to the question page? It would increase the initial page load but then there would be less of a delay when starting the application.

Right, I'll change that